### PR TITLE
update icon for "Controller Options" in Settings

### PR DESCRIPTION
**Pull request by cvan**
image

P.S. I noticed I need to alter the height to make it more consistent with the height of the other icons. will fix before merging.

### CODE REVIEW

**Comment by philip-lamb** [comment]
I think we'll want to selectively load this icon based on the platform.

**Comment by cvan** [comment]
Yep, on it. Have fixed it locally. Will push tomorrow, Friday, and with @thenadj's Go models too. Thanks!

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/SettingsWidget.java [hunk 1]
```diff
@@ -194,6 +195,26 @@ private void initialize(Context aContext) {
...
+            break;
+          case DeviceType.OculusQuest:
+            Log.d(LOGTAG, "Device type: Oculus Quest");
+            controllerOptionsButton.setVisibility(View.GONE);
+            controllerOptionsButton = findViewById(R.id.controllerOptionsButtonOculus6dof);
+            controllerOptionsButton.setVisibility(View.VISIBLE);
+            break;
+          default:
+            Log.d(LOGTAG, "Device type: Default");
+            controllerOptionsButton = findViewById(R.id.controllerOptionsButton);
```
Comment (cvan) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/SettingsWidget.java]: can remove

File: app/src/main/res/layout/settings.xml [hunk 2]
```diff
@@ -156,11 +156,29 @@
                         <org.mozilla.vrbrowser.ui.views.HoneycombButton
                             android:id="@+id/controllerOptionsButton"
                             style="?attr/honeycombButtonStyle"
+                            android:layout_alignTop="@+id/second_row"
```
Comment (bluemarvin) [inline app/src/main/res/layout/settings.xml]: This should be `android:visibility="gone"` instead?

Comment (cvan) [reply app/src/main/res/layout/settings.xml]: yeah, must've changed it visually in the View

**Pull request merged**